test/es-module/test-esm-get-source-loader.mjs

```diff
@@ -0,0 +1,6 @@
+// Flags: --experimental-loader ./test/fixtures/es-module-loaders/get-source.mjs
+/* eslint-disable node-core/require-common-first, node-core/required-modules */
+import assert from 'assert';
+import { message } from '../fixtures/es-modules/message.mjs';
+
+assert.strictEqual(message, 'WOOHOO!');
```

Comment:
this hook should not include format as an argument/if it does it needs tests where the return value differs from the argument